heron/tools/apiserver/src/java/com/twitter/heron/apiserver/Constants.java

```diff
@@ -0,0 +1,22 @@
+//  Copyright 2017 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.apiserver;
+
+public class Constants {
+
+  static final int DEFAULT_PORT = 9000;
```

Comment:
I wanted something different until we collapse it so it doesn't interfere with the tracker and ui default ports.